lib/services/ca.go

```diff
@@ -130,7 +130,7 @@ func (ca *CertAuthority) Checkers() ([]ssh.PublicKey, error) {
 	for _, keyBytes := range ca.CheckingKeys {
 		key, _, _, _, err := ssh.ParseAuthorizedKey(keyBytes)
 		if err != nil {
-			return nil, trace.Wrap(err)
+			return nil, trace.Errorf("invalid authority public key (len=%d): %v", len(keyBytes), err)
```

Comment:
can you revendor `trace` and start using `trace.Wrap(err, "user friendly error here")`